pkg/utils/pod/pod_utils.go

```diff
@@ -91,25 +92,31 @@ func getPodConditionFromList(conditions []corev1.PodCondition, conditionType cor
 	return -1, nil
 }
 
-func addEnvVarIfNotExists(c *corev1.Container, e corev1.EnvVar) {
-	for _, env := range c.Env {
-		if env.Name == e.Name {
-			return
+func addEnvVarsIfNotExists(c *corev1.Container, e ...corev1.EnvVar) {
+	for _, newEnv := range e {
+		exists := false
+		for _, env := range c.Env {
+			if env.Name == newEnv.Name {
+				exists = true
+				break
+			}
+		}
+		if !exists {
+			c.Env = append([]corev1.EnvVar{newEnv}, c.Env...)
```

Comment:
could you do a favor to add a comment here to mention that we need to ensure the env vars that gets populated by lws are always at the front of other env vars

Reply:
Here I split the first variable and the e slice and ensured the slice order

Reply:
And add some comments